api/api.go

```diff
@@ -0,0 +1,270 @@
+package api
+
+import (
+	"net/http"
+
+	"encoding/json"
+	"errors"
+	"github.com/ONSdigital/dp-import-api/api-errors"
+	"github.com/ONSdigital/dp-import-api/models"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/gorilla/mux"
+)
+
+const internalError = "Internal server error"
+
+// ImportAPI - A restful API used to manage importing datasets to be published
```

Comment:
So all the other comments:
https://golang.org/doc/effective_go.html#commentary

`The first sentence should be a one-sentence summary that starts with the name being declared.`

So `CreateImportAPI creates an instance of the api with all the routes configured` etc.